Openshift-EE/pipelines/Cassandra/stages/4-chaos/M0NH-jiva-app-kill/jiva-app-kill.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+set -x
+
+pod() {
+sshpass -p $pass ssh -o StrictHostKeyChecking=no $user@$ip -p $port 'cd e2e-openshift && bash Openshift-EE/pipelines/Cassandra/stages/4-chaos/M0NH-jiva-app-kill/jiva-app-kill node '"'$CI_JOB_ID'"'' '"'$CI_PIPELINE_ID'"' '"'$CI_COMMIT_SHA'"'
+}
+
+node() {
+
+#Gitlab job id obtain from the gitlab env ($CI_JOB_ID).
+job_id=$1
+
+# Gitlab pipeline id obtained from gitlab env ($CI_PIPELINE_ID).
+pipeline_id=$2 
+
+#Gitlab commit id Obtained fron gilab env ($CI_COMMIT_SHA).
+commit_id=$3 
+
+#test case ID.
+case_id=M0NH
+
+#github token to push the test result into github mayadata-io/e2e-openshift repository.
+#This token is set as an env in ~/.profile file in the test cluster.
+source ~/.profile
+gitToken=$github_token
+
+time="date"
+current_time=$(eval $time)
+
+present_dir=$(pwd)
+echo $present_dir
+
+bsah Openshift-EE/utils/pooling jobname:cstor-volume-mgmt-kill
+bash Openshift-EE/utils/e2e-cr jobname:jiva-app-kill jobphase:Running init_time:"$current_time" jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+
+
+################
+# LitmusBook 1 #
+################
+
+#### Generating test name using test case name
+
+run_id="jiva";test_name=$(bash Openshift-EE/utils/generate_test_name testcase=application-pod-failure metadata="")
+echo $test_name
+
+cd litmus
+
+# copy the content of chaos `run_litmus_test.yml` litmusbook into a different file to update the test specific parameters.
+
+cp experiments/chaos/app_pod_failure/run_litmus_test.yml jiva_app_kill.yml
+
+# Update the environmental variables in litmus job spec.
+
+: << EOF
+---------------------------------------------------------------------------------------------------------------------
+| specAttribute      | kind  | default value                        | test specifc value                             |
+---------------------------------------------------------------------------------------------------------------------|
+| appLabel           | env   | app=jenkins-app                      | app=cassandra-jiva                             | 
+| Litmus Job name    | name  | generateName: application-pod-failure| generateName: application-pod-failure-jiva     |
+| Litmus job label   | label | name: application-pod-failure        | name: application-pod-failure-jiva             |
+| appNamespace       | env   | app-jenkins-ns                       | app-cass-ns-jiva                               | 
+----------------------------------------------------------------------------------------------------------------------
+EOF
+
+sed -i -e 's/value: '\''app=jenkins-app'\''/value: '\''app=cassandra-jiva'\''/g' \
+-e 's/generateName: application-pod-failure/generateName: application-pod-failure-jiva/g' \
+-e 's/name: application-pod-failure/name: application-pod-failure-jiva/g' \
+-e 's/value: app-jenkins-ns/value: app-cass-ns-jiva/g' app_kill_jiva.yml
+
+sed -i '/command:/i \
+- name: RUN_ID\
+value: '"$run_id"'\
+' jiva_app_kill.yml
```

Comment:
@w3aman check if this sed command is working?

Reply:
I've modified the sed command. Now it is working...!!!